ArmstrongPy.py

```diff
@@ -1,13 +1,31 @@
 # Python program to check if the number is an Armstrong number or not
-num = int(input("Enter a number: "))
-sum = 0
+class ArmStrong():
+ def Armstrong(self):
+    num = 371
+    sum = 0
+    temp = num
+    while temp > 0:
+        digit = temp % 10
+        sum += digit ** 3
+        temp //= 10
+    if num == sum:
+        print(num, "is an Armstrong number")
+    else:
+        print(num, "is not an Armstrong number")
+<<<<<<< HEAD
```

Comment:
work on this HEAD

Reply:
@amitsahu8 Rework done, Kindly verify it